### ISSUE TITLE
What version of the code should one be using?

### ISSUE THREAD

**Issue by sanjayankur31** [lemberg/connfa-integration-server#37]
Hi!

I've been trying to get the server to run for a few hours now, and have tried the commit for version 2.0.0, 2.1.0 and master---they all fail with different errors. What version should one be using? Should one use master and report bugs against it?

Thanks!

**Comment by sanjayankur31** [comment]
On master, I get this:

```
php artisan migrate
Migration table created successfully.

                                                                                                                                                                        
  [Illuminate\Database\QueryException]                                                                                                                                  
  SQLSTATE[HY000]: General error: 1832 Cannot change column 'event_id': used in a foreign key constraint 'schedule_event_event_id_foreign' (SQL: alter table `schedule  
  _event` add primary key `schedule_event_schedule_id_event_id_primary`(`schedule_id`, `event_id`))                                                                     
                                                                                                                                                                        

                                                                                                                                            
  [PDOException]                                                                                                                            
  SQLSTATE[HY000]: General error: 1832 Cannot change column 'event_id': used in a foreign key constraint 'schedule_event_event_id_foreign'  
```

but I don't get it on 2.0.0. (There, I get an error with the next command)

**Comment by sanjayankur31** [comment]
This is the complete stack trace:

```
Next Illuminate\Database\QueryException: SQLSTATE[HY000]: General error: 1832 Cannot change column 'event_id': used in a foreign key constraint 'schedule_event_event_id_foreign' (SQL: alter table `schedule_event` add primary key `schedule_event_schedule_id_event_id_primary`(`schedule_id`, `event_id`)) in /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Connection.php:680
Stack trace:
#0 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Connection.php(636): Illuminate\Database\Connection->runQueryCallback('alter table `sc...', Array, Object(Closure))
#1 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Connection.php(409): Illuminate\Database\Connection->run('alter table `sc...', Array, Object(Closure))
#2 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Schema/Blueprint.php(83): Illuminate\Database\Connection->statement('alter table `sc...')
#3 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Schema/Builder.php(229): Illuminate\Database\Schema\Blueprint->build(Object(Illuminate\Database\MySqlConnection), Object(Illuminate\Database\Schema\Grammars\MySqlGrammar))
#4 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Schema/Builder.php(148): Illuminate\Database\Schema\Builder->build(Object(Illuminate\Database\Schema\Blueprint))
#5 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(221): Illuminate\Database\Schema\Builder->create('schedule_event', Object(Closure))
#6 /home/asinha/public_html/connfa-integration-server/database/migrations/2017_03_28_081510_create_schedules_events_table.php(26): Illuminate\Support\Facades\Facade::__callStatic('create', Array)
#7 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Migrations/Migrator.php(147): CreateSchedulesEventsTable->up()
#8 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Migrations/Migrator.php(117): Illuminate\Database\Migrations\Migrator->runUp('2017_03_28_0815...', 1, false)
#9 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Migrations/Migrator.php(86): Illuminate\Database\Migrations\Migrator->runMigrationList(Array, Array)
#10 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Database/Console/Migrations/MigrateCommand.php(76): Illuminate\Database\Migrations\Migrator->run('/home/asinha/pu...', Array)
#11 [internal function]: Illuminate\Database\Console\Migrations\MigrateCommand->fire()
#12 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#13 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Console/Command.php(169): Illuminate\Container\Container->call(Array)
#14 /home/asinha/public_html/connfa-integration-server/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Console/Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /home/asinha/public_html/connfa-integration-server/vendor/symfony/console/Application.php(791): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /home/asinha/public_html/connfa-integration-server/vendor/symfony/console/Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /home/asinha/public_html/connfa-integration-server/vendor/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /home/asinha/public_html/connfa-integration-server/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /home/asinha/public_html/connfa-integration-server/artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 {main}

```

**Comment by yuriymyrosh** [comment]
@sanjayankur31  hi, we had just released v2.0.1 with bug fixes, fix for this issue is already there, you can try to use tag 2.0.1 or master branch
